transform/transformers/CORATransformer.py

```diff
@@ -0,0 +1,332 @@
+#!/usr/bin/env python
+#   coding: UTF-8
+
+from collections import OrderedDict
+import enum
+import os.path
+import re
+import sys
+
+from transform.transformers.CSTransformer import CSTransformer
```

Comment:
I understand that but CSTransformer stands for Common Software which this repo should have no knowledge of. It would be much clearer to take the create zipfiles method out of CS and place it directly in the CoraTransformer. This way there is not a several hundred line file in the repo with only one method being used, and the inheritance can be removed so the definition becomes more explicit. CSTransformer absolutely cannot continue to exist in the repo.